pkg/url/url_test.go

```diff
@@ -1490,19 +1522,28 @@ func TestPush(t *testing.T) {
 			existingEnvInfoURLs: []envinfo.EnvInfoURL{
 				{
 					Name: "example",
-					Port: 8080,
 					Kind: envinfo.ROUTE,
 				},
 			},
-			endpintMap: map[int32]versionsCommon.Endpoint{
-				8080: versionsCommon.Endpoint{
-					Name:       "example",
-					TargetPort: 8080,
-					Secure:     false,
-					Exposure:   "internal",
+			containerComponents: []versionsCommon.DevfileComponent{
```

Comment:
this case seems to be the same case as above as suggested from the name of the case

Reply:
Good catch. Deleted the duplicate test case